code/game/objects/items/mop.dm

```diff
@@ -18,18 +18,23 @@
 	var/mopspeed = 15
 	force_string = "robust... against germs"
 	var/insertable = TRUE
+	///If the mop is braced to use special moves
 	var/braced = FALSE
+	///Skill requirement for mopping while walking
+	var/braced_skillreq_walk = SKILL_LEVEL_JOURNEYMAN
+	///Skill requirement for mopping while running
+	var/braced_skillreq_run = SKILL_LEVEL_MASTER
 
 /obj/item/mop/Initialize()
 	. = ..()
 	create_reagents(mopcap)
 
 /obj/item/mop/attack_self(mob/user)
-	if(user.mind.get_skill_level(/datum/skill/cleaning) >= SKILL_LEVEL_JOURNEYMAN)
+	if(user.mind.get_skill_level(/datum/skill/cleaning) >= braced_skillreq_walk)
```

Comment:
The use of defines was ok. I meant that you should only call get_skill_level once.

Reply:
OOH I get what you mean now, duh. I thought it was odd. Think I'm gonna keep the skill levels a variable anyway, it might come in handy